### PR TITLE
cocacolaHBC & readme update for old repo users

### PR DESCRIPTION
**Pull request by Codetamer333**

### CODE REVIEW

File: sites/cocacola.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+"use strict";
+const scraper = require("../peviitor_scraper.js");
+const uuid = require("uuid");
```
Comment (lalalaurentiu) [inline sites/cocacola.js:3]: Uuid is no longer needed
```suggestion
```

File: sites/cocacola.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const url = "https://careers.coca-colahellenic.com/ro_RO/careers/SearchJobs/?1293=%5B6003%5D&1293_format=2880&listFilterMode=1&projectRecordsPerPage=50";
+
+const company = { company: "Coca Cola HBC" };
```
Comment (lalalaurentiu) [inline sites/cocacola.js:7]: company name must not contain spaces

```suggestion
const company = { company: "CocaColaHBC" };
```

File: sites/cocacola.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      const job_title = job.find("a").text.trim(); 
+      const job_link = job.find("a").attrs.href;
+      const cityArray = job.find("span", { class: "list-item" }).text.trim().match(regex);
+      let city = ""; 
+      if (cityArray && cityArray.length >= 2) {
+         city = cityArray[1];
+        console.log(city);
+    }else{
+         city = "Multiple Locations";
+        console.log(city);
```
Comment (lalalaurentiu) [inline sites/cocacola.js:27]: Delete this console.log

```suggestion
```

File: sites/cocacola.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    const jobs = soup.findAll("div",{ class: "article__header__text" });
+    jobs.forEach((job) => {
+      const id = uuid.v4();
+      const job_title = job.find("a").text.trim(); 
+      const job_link = job.find("a").attrs.href;
+      const cityArray = job.find("span", { class: "list-item" }).text.trim().match(regex);
+      let city = ""; 
+      if (cityArray && cityArray.length >= 2) {
+         city = cityArray[1];
+        console.log(city);
```
Comment (lalalaurentiu) [inline sites/cocacola.js:24]: Delete this console.log

```suggestion
```

File: sites/cocacola.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+let finalJobs = [];
+const regex =/\((.*?)\)/;
+const apiKey = process.env.KNOX
+const s = new scraper.Scraper(url);
+
+s.soup
+  .then((soup) => {
+    const jobs = soup.findAll("div",{ class: "article__header__text" });
+    jobs.forEach((job) => {
+      const id = uuid.v4();
```
Comment (lalalaurentiu) [inline sites/cocacola.js:17]: Uuid is no longer needed
```suggestion
```

File: sites/cocacola.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      let city = ""; 
+      if (cityArray && cityArray.length >= 2) {
+         city = cityArray[1];
+        console.log(city);
+    }else{
+         city = "Multiple Locations";
+        console.log(city);
+    }
+      finalJobs.push({
+        id: id,
```
Comment (lalalaurentiu) [inline sites/cocacola.js:30]: Uuid is no longer needed
```suggestion
```

**Review by lalalaurentiu — CHANGES_REQUESTED** [review]

File: sites/cocacola.js (hunk 1 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

**Review by lalalaurentiu — APPROVED** [review]

File: sites/cocacola.js (hunk 2 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

File: sites/cocacola.js (hunk 3 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

File: sites/cocacola.js (hunk 4 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

File: sites/cocacola.js (hunk 5 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

File: sites/cocacola.js (hunk 6 above)
Comment (Codetamer333) [reply sites/cocacola.js]: Done

**Comment by Codetamer333** [comment]
I have made all the required changes

**Pull request merged**